assets/css/_crowding_diagram.scss

```diff
@@ -0,0 +1,82 @@
+$color-empty-text: $color-component-medium;
+$color-empty-icon: $color-secondary-light;
+$color-not-crowded: #5cbf95;
+$color-some-crowding: #fca400;
+$color-crowded: #e90d0d;
```

Comment:
Since these are global variables, perhaps they should be namespaced, like `$color-crowding-crowded`/`$color-crowding-empty-icon` or something?